sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -300,11 +300,11 @@ private void injectLocalizedResources(JSONObject configJson, Context context) {
   /**
    * Set the default audio language from skin.json provided the language exists.
    *
-   * @param audioLanguage The part of the config that define the default audio language.
+   * @param audioParams The part of the config that define the default audio language.
    */
-  private void setDefaultAudioLanguage(JSONObject audioLanguage) {
+  private void setDefaultAudioLanguage(JSONObject audioParams) {
     try {
-      String language = audioLanguage.getString("audioLanguage");
+      String language = audioParams.getJSONObject("audio").getString("audioLanguage");
```

Comment:
Would you mind to add null check?
JSONObject audioParams = audioParams.getJSONObject("audio");
if (audioParams != null) {
...
}